### DISCUSSION TITLE
Dedicated VS Code extension

### DISCUSSION THREAD

**Discussion by brettcannon** [akaihola/darker#409] [Ideas]
In case you're interested in having your own VS Code extension, https://code.visualstudio.com/api/advanced-topics/python-extension-template should make it easy (as in "an afternoon or less" easy) to create an extension which ships with Darker in-box and that you control (see the [Black](https://github.com/microsoft/vscode-black-formatter) and [isort](https://github.com/microsoft/vscode-isort/) extensions as examples).

**Comment by akaihola** [comment]
Wow, thanks @brettcannon, I do want to look into that!
